examples/common/env.sh

```diff
@@ -80,5 +80,7 @@ alias mysql="command mysql --no-defaults -h 127.0.0.1 -P 15306"
 alias vtctldclient="command vtctldclient --server localhost:15999"
 
 # Make sure aliases are expanded in non-interactive shell
-shopt -s expand_aliases
+if [[ "${SHELL##*/}" == "bash" ]]; then
```

Comment:
Turns out that `$SHELL` is the current users's default but not necessarily what's actually being used (to execute commands):
```
❯ bash -c 'echo $SHELL; echo ${0}'
/bin/zsh
bash

❯ /bin/bash -c 'echo $SHELL; echo ${0}'
/bin/zsh
/bin/bash

❯ /bin/bash -c 'echo $SHELL; echo ${0##*/}'
/bin/zsh
bash
```

That's the reason for the test failures, as the aliases are not getting expanded. We can instead do:
```
if [[ "${0##*/}" == "bash" ]]; then
```

The local examples then run fine / as expected locally for me  — using a bash shell to run `./101_initial_cluster.sh` — with this change.

Reply:
I take that back... there's still some missing piece ...

Reply:
OK, so the problem with that is that $0 becomes the command run in the parent shell when starting sub-shells like we do in the scripts. 

So... the winner is checking for the existence of a bash only variable:
```
if [[ -n $BASH ]]; then
    shopt -s expand_aliases
fi
```

Reply:
THAT I have successfully works locally 😄

Reply:
@mattlord, I am running `101_initial_cluster.sh` with the code in this PR from zsh, after modifying `env.sh` to print the result of `${SHELL##*/}` very early on, I see `zsh`. I am not sure I fully understand your comment saying we should use `${BASH}` instead

Reply:
Thanks for investigating! I also tested on mac and ubuntu 24.04 and works as it should.